src/Tribe/API/Contact_Property_Group.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Tribe\HubSpot\API;
+
+use SevenShores\Hubspot\Factory;
+
+/**
+ * Class Contact_Property_Group
+ *
+ * @package Tribe\HubSpot\API
+ */
+class Contact_Property_Group {
+
+	/**
+	 * @var string
+	 */
+	protected $display_name = 'Event Tickets';
+
+	/**
+	 * @var string
+	 */
+	public $group_name = 'event_tickets';
+
+	/**
+	 * Setup Hooks for Contact_Property_Group
+	 *
+	 * @since 1.0
+	 *
+	 */
+	public function hook() {
+
+		add_action( 'tribe_hubspot_authorize_site', [ $this, 'create' ], 20 );
+	}
+
+	/**
+	 * Create HubSpot Group if not Already Created
+	 *
+	 * @since 1.0
+	 *
+	 */
+	public function create() {
+
+		if ( $hubspot_api_group = $this->has_group() ) {
+			return;
+		}
+
+		/** @var \Tribe\HubSpot\API\Connection $hubspot_api */
+		$hubspot_api = tribe( 'tickets.hubspot.api' );
+		if ( ! $access_token = $hubspot_api->is_ready() ) {
+			return false;
```

Comment:
Return types are off here, if you are planning on returning true/false then define that in phpdoc and make sure it's consistent throughout the method. Otherwise this can go to 
```suggestion
			return;
```

Reply:
Removed the return false.